test/scripts/withdraw.js

```diff
@@ -18,6 +19,8 @@ const { waitForNSeconds, execTransaction } = require("../../scripts/utils/intern
 const prepareWithdraw = require("../../scripts/wrapper/withdraw")(web3, artifacts)
 const { toErc20Units, fromErc20Units } = require("../../scripts/utils/printing_tools")
 
+const bnMaxUint256 = new BN(2).pow(new BN(256)).subn(1)
```

Comment:
ubernit: why not reuse the one from printing tools?

Reply:
Agree that this is probably already somewhere in the project, exported and ready for import.

Reply:
I thought about it, but since this is a test and that is part of the logic of the code I think it makes sense to define them independently. I don't have a strong opinion on that, I'm fine with importing it from the printing tools. Or to add this constant to `test_utils.js` and import it from there.